ethereum/contracts/nft/NFTBridgeSetters.sol

```diff
@@ -3,9 +3,11 @@
 
 pragma solidity ^0.8.0;
 
+import "../ShutdownSwitch.sol";
+
 import "./NFTBridgeState.sol";
 
-contract NFTBridgeSetters is NFTBridgeState {
+contract NFTBridgeSetters is NFTBridgeState, ShutdownSwitch {
```

Comment:
it's a nitpick, but semantically I think this class should be extended by the `NFTBridgeGovernance` class instead (same for `BridgeGovernance`)

Reply:
The problem is that BridgeSetup::setup() needs access to the setUp method in ShutdownSwitch, and BridgeSetUp inherits from BridgeSetters, not BridgeGovernance. BridgeSetters is the common root of both Bridge and BridgeSetup.